dev-docs/source/ReleaseChecklist.rst

```diff
@@ -153,21 +153,38 @@ language, layout, and collecting images.
 Beta Testing Begins
 ###################
 
-*  Switch to manual handling of release notes by running the release_editor.py script using the
-   `release editor helper tool
-   <https://github.com/mantidproject/mantid/blob/main/tools/ReleaseNotes/release_editor.py>`_
-   and open a pull request to put them on ``release-next``. The script copies all of the separate release notes under the correct heading of their upper level file, e.g. framework.rst, and moves
-   the original release notes into a 'Used' folder. This makes it easier for the Release Editor to see which notes have been copied over and which haven't and prevents losing notes or merge conflicts.
+* Initial amalgamation of the the release notes:
 
-.. code-block:: bash
+  * ``git pull`` on ``release-next``.
+  * Create a new branch using the `Mantid Git Workflow guidance <https://developer.mantidproject.org/GitWorkflow.html#new-branches>`_.
+  * Navigate to your Mantid 'build' directory and open ``command-prompt.bat``.
+  * In the new command prompt, navigate to the `release_editor.py script <https://github.com/mantidproject/mantid/blob/main/tools/ReleaseNotes/release_editor.py>`_ and run, parsing the correct version number. The script copies all of the separate release notes under the correct heading of their upper level file, e.g. ``framework.rst``, and moves the original release notes into a 'Used' directory.
+
+    .. code-block:: bash
+
+      python release_editor.py --release 6.5.0
+
+  * Check the script has run correctly by checking all individual rst files have been moved into their respective 'used' directories.
```

Comment:
Maybe it would be good to mention the unused release note finder tool here? I know you explain further below but perhaps suggest the tool could be used here (see below for further details) or something to that effect?